gnovm/cmd/gno/util.go

```diff
@@ -86,9 +86,14 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 				}
 				visited[parentDir] = true
 
-				// cannot use path.Join or filepath.Join, because we need
-				// to ensure that ./ is the prefix to pass to go build.
-				pkg := "./" + parentDir
+				pkg := parentDir
```

Comment:
would be better also to rename this pkgDir
since pkg is ambiguous, pkgpath or dirpath